### PR TITLE
[DOCS-7624] Update Supported Platforms in Search modules

### PR DESCRIPTION
**Pull request by Prosune**

### CODE REVIEW

**Review by anxumalo — CHANGES_REQUESTED** [review]
Please see inline comments - I also expected to see changes in the ACS Supported platforms pages

File: insight-engine/latest/support/index.md
[the remark this answers is not in the document]
Comment (Prosune) [reply insight-engine/latest/support/index.md:1]: These versions appear to be included. I could not see them initially but when I expanded, I could see them.

File: search-enterprise/latest/support/index.md [hunk 1]
```diff
@@ -6,9 +6,8 @@ The following are the supported platforms for Search Enterprise 4.0.x:
 
 | Version | Notes |
 | ------- | ----- |
-| Content Services 23.1.x | Including ActiveMQ, Alfresco Transform Service, and the database |
-| | Check the [Alfresco Content Services Supported platforms]({% link content-services/latest/support/index.md %}) page for specific versions of the individual components. |
-| | |
```
[the remark this answers is not in the document]
Comment (Prosune) [reply search-enterprise/latest/support/index.md:11]: These versions appear to be included. I could not see them initially but when I expanded, I could see them.

File: search-services/latest/support/index.md
[the remark this answers is not in the document]
Comment (Prosune) [reply search-services/latest/support/index.md:1]: These versions appear to be included. I could not see them initially but when I expanded, I could see them.

File: search-enterprise/latest/support/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Prosune) [reply search-enterprise/latest/support/index.md:11]: Done

**Review by anxumalo — APPROVED** [review]

**Pull request merged**